### PR TITLE
feat: change of the config location mentioned in #57

### PR DESCRIPTION
**Pull request by Senna46**
# Change of the config location mentioned in #57

@YasunoriMATSUOKA 
Too many modifications were made, so they are implemented in a separate PR.
After approving this PR, please also #57

### CODE REVIEW

File: editions/ununifi/launch/ununifi-8-private-test/docker-compose.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 version: '3'
 services:
-  ununifi-explorer:
-    container_name: ununifi-explorer
-    image: ghcr.io/ununifi/explorer:test
+  telescope-ununifi:
+    container_name: telescope-ununifi
+    image: ghcr.io/ununifi/telescope-ununifi:test
```
Comment (YasunoriMATSUOKA) [inline editions/ununifi/launch/ununifi-8-private-test/docker-compose.yml:5]: I think this is strange.

File: editions/ununifi/launch/ununifi-8-private-test/docker-compose.yml [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
     volumes:
       - ./config.js:/usr/share/nginx/html/assets/config.js
+      - ./config.js:/usr/share/nginx/html/ununifi/assets/config.js
```
Comment (YasunoriMATSUOKA) [inline editions/ununifi/launch/ununifi-8-private-test/docker-compose.yml:8]: I think this is not necessary.

**Review by YasunoriMATSUOKA — CHANGES_REQUESTED** [review]
@Senna46 
I think telescope-ununifi related implementation should not exists.
Can you check that?
Thank you 🙏

**Comment by Senna46** [comment]
@YasunoriMATSUOKA 
Portal and Explorer files were split and script files were rewritten.
Please be careful when deploying and creating Docker images.

**Comment by Senna46** [comment]
@YasunoriMATSUOKA 
Separate files required for deployment are placed in Portal and Explorer.
The same Config can be used for both.

**Review by YasunoriMATSUOKA — APPROVED** [review]
LGTM

**Pull request merged**